rfcs/disable_chrome_stability.md

```diff
@@ -0,0 +1,38 @@
+# RFC 131: Disable wpt-chrome-dev-stability runs
+
+## Summary
+
+The wpt-chrome-dev-stability job on WPT fails quite frequently and it's
+impacting developer productivity. The job should be disabled since it is not
+providing enough value or at least change to non-blocking.
+
+## Details
+
+Stability jobs in principle help us ensure that new and modified tests aren't
+flaky, wasting engineering time down the road as they block landing unrelated
+PRs. We currently run stability jobs for Firefox and Chrome, but not when
+[merging export PRs from their own repositories](https://github.com/web-platform-tests/wpt/issues/29737).
+The reason is that browser engineers usually have better tools in their
```

Comment:
I don't think we should make claims about vendor CIs having better tooling in general. The reason we don't run them for PRs introduced by those browsers isn't that we expect they have _better_ tooling, but that we assume that a change which was considered stable enough to land in a vendor repo is also stable enough to land upstream, or at least that if it does cause flake in the vendor repo it will be fixed. In addition it's unusually likely that patches from vendor repos _add_ flaky tests at the same time as fixing the underlying issue. That's fine in the vendor CI where the build running the test will have the code changes, but since the builds we are running won't get those changes until they reach the relevant experimental channel, the added tests are likely to be unstable in GitHub CI.

Reply:
I don't think I am making any strong claims here, I said "_usually_ have better tools". Your comment suggests that _something_ makes doing the work downstream "better", but doesn't clearly identify that is. I tried to relax it further to "usually have better tools or processes", I hope that is satisfactory.